src/ts/components/subtitleoverlay.ts

```diff
@@ -256,18 +280,20 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
   }
 
   enablePreviewSubtitleLabel(): void {
-    this.previewSubtitleActive = true;
     if (!this.subtitleManager.hasCues) {
+      this.previewSubtitleActive = true;
       this.subtitleContainerManager.addLabel(this.previewSubtitle);
       this.updateComponents();
       this.show();
     }
   }
 
   removePreviewSubtitleLabel(): void {
-    this.previewSubtitleActive = false;
-    this.subtitleContainerManager.removeLabel(this.previewSubtitle);
-    this.updateComponents();
+    if (this.previewSubtitleActive) {
```

Comment:
nit: That way you are not the owner of all lines, but only the updated IF clause
```suggestion
    if (!this.previewSubtitleActive) {
        return;
    }
```

Reply:
Changed in https://github.com/bitmovin/bitmovin-player-ui/pull/378/commits/144549952d0db2a91607a93623db00df517d7dcb